src/cbc_sdk/platform/policies.py

```diff
@@ -1031,10 +1032,28 @@ def preview_rank_change(self, new_rank):
                             where _N_ is the total number of policies in the organization.
 
         Returns:
-            list[PolicyRankChangePreview]: A list of objects containing data previewing the policy changes.
+            list[DevicePolicyChangePreview]: A list of objects containing data previewing the policy changes.
         """
         return Policy.preview_policy_rank_changes(self._cb, [(self._model_unique_id, new_rank)])
 
+    def preview_add_policy_override(self, devices):
```

Comment:
Not sure this is necessary only remove policy override applies

Reply:
This previews what happens to a series of devices if you add _this_ policy as an override for them.  The results could be different for each device, as to _which_ policy is being overridden.

However, if you think this is also redundant, this can be erased with little problem.

Reply:
I guess I'm confused when would a policy override not apply as it should out rank any policy based position

Reply:
Well, let's say one device has policy Alpha with priority 2, and another device has policy Bravo with priority 3. If you then have policy Charlie with priority 4 override both devices, you would get two separate preview notifications, both having Charlie as the "new" policy, but one with Alpha as "current," and one with Bravo as "current."